modules/javafx.graphics/src/main/java/com/sun/javafx/application/PlatformImpl.java

```diff
@@ -1061,4 +1061,12 @@ private static void checkHighContrastThemeChanged(Map<String, Object> preference
         }
     }
 
+    private static native void _overrideNativeWindowHandle(Class lwFrameWrapperClass,
+                                                           Object frame,
+                                                           long handle, Runnable closeWindow);
+
+    public static void overrideNativeWindowHandle(Class lwFrameWrapperClass, Object frame,
+                                                  long handle, Runnable closeWindow) {
+        _overrideNativeWindowHandle(lwFrameWrapperClass, frame, handle, closeWindow);
+    }
```

Comment:
`PlatformImpl` is in a package / subsystem that doesn't have any native methods, and I'd prefer to keep it that way.

I recommend moving these methods to glass, probably `com.sun.glass.ui.Application`, since that will be a simple matter to cut / paste the methods from here to there.